eng/feeds/arm/tspconfig.yaml

```diff
@@ -6,6 +6,39 @@ options:
     emitter-output-dir: "{project-root}/.."
     azure-resource-provider-folder: "resource-manager"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-csharp":
+    package-dir: {package-name}
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
```

Comment:
@msyyc could you verify?

Reply:
Not sure what is the meaning of `normalizePackageName` and `ServiceNamespace` and you can refer to the template https://github.com/Azure/azure-rest-api-specs/blob/ab67c148ec716a0d0075770742d54468f128c72e/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L18-L23

Reply:
I copied it from the data plane template: https://github.com/Azure/typespec-azure/blob/9438df7aad912cdc5af559558cbae16c7431ea80/eng/feeds/data-plane/tspconfig.yaml#L15
What do you recommend the generic value for `package-dir` should be?